draft-ietf-httpbis-priority.md

```diff
@@ -682,6 +686,14 @@ basic recommendations for implementations.
 Clients cannot depend on particular treatment based on priority signals. Servers
 can use other information to prioritize responses.
 
+It is generally beneficial for the server to maximize its use of available
+bandwidth when serving concurrent requests. Bandwidth utilization for any given
+connection is an output of scheduling that can be used in a feedback loop for
+subsequent scheduling decisions. For example, a server that prioritizes sending
+response data for a resource that it can only read slowly might find bandwidth
+is under-utilized. The server could redistribute the available bandwidth to
+other requests.
```

Comment:
I think we can drop this paragraph If we adopt something similar to what I have proposed above.

I get the intent of this addition, but to me it is a bit hard to digest. I have difficulty understanding the second sentence. Also, I'm not sure if "re-"distribute found in the third sentence is precise. The text on `urgency` as a suggesting the precedence between requests, therefore, "re-"distribution should be unnecessary.

Another argument against this clause is that HTTP servers are required to serve responses in timely manner regardless of the existence of a prioritization mechanism and therefore that this type of recommendation is not strictly necessary.

Reply:
removed the para